docs/release-info/release-notes.md

```diff
@@ -57,9 +57,8 @@ For tips about how to write a good release note, see [Release notes](https://git
 
 As part of the continued improvements to Druid, we are deprecating certain features and behaviors in favor of newer iterations that offer more robust features and are more aligned with standard ANSI SQL. Many of these new features have been the default for new deployments for several releases.
 
-The following features are deprecated, and we plan to remove support as early as Druid 36.0.0:
+The following features are deprecated, and we plan to remove support in Druid 32.0.0:
```

Comment:
should we leave this as 'as early as' just in case we don't get it done in time and also the upgrade notes still uses that phrasint